src/pages/NewBeer.jsx

```diff
@@ -0,0 +1,109 @@
+import Header from '../components/Header';
+import { useState } from 'react';
+import '../pages/NewBeer.css';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+function NewBeer() {
+  const [name, setName] = useState('');
+  const [tagline, setTagLine] = useState('');
+  const [description, setDescription] = useState('');
+  const [firstBrewed, setFirstBrewed] = useState('');
+  const [brewersTips, setBrewersTips] = useState('');
+  const [attenuationLevel, setAttenuationLevel] = useState(0);
+  const [contributedBy, setContributedBy] = useState('');
+
+  const navigate = useNavigate();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    // Creating the body for the POST request
+    // using the object property value shorthand
+    // values are coming from the state variable
+    const body = { name, tagline, description, first_brewed: firstBrewed, brewers_tips: brewersTips, attenuation_level: attenuationLevel, contributed_by: contributedBy };
+    console.log(body);
+
+    axios
+      .post('https://ih-beers-api2.herokuapp.com/beers/new', body)
+      .then((response) => {
+        console.log(response.data);
+        setName('');
+        setTagLine('');
+        setDescription('');
+        setFirstBrewed('');
+        setBrewersTips('');
+        setAttenuationLevel(0);
+        setContributedBy('');
+
+        navigate('/');
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  return (
+    <div>
+      <div className="header">
+        <Header />
+      </div>
+      <form onSubmit={handleSubmit}>
+        <div className="text-align-left form-container">
+          <div className="input-container mb-25">
+            <label className="d-block" htmlFor="name">
+              Name
+            </label>
+            <input type="text" name="name" onChange={(e) => setName(e.target.value)} value={name} />
```

Comment:
I think you were using the `name` attribute to connect the label and the input (if that wasn't the case, I would suggest adding that one) the `htmlFor` connects through the `id` => That's why I usually prefer to wrap the `<input>` with the `<label>` so it creates that connection without using an `id`

I would add the `required` so the user would be blocked from sending the request without the info that's needed for the API. It would also add some feedback for the user

```suggestion
            <input type="text" id="name" onChange={(e) => setName(e.target.value)} value={name} required />
```